app/templates/components/public/ticket-list.hbs

```diff
@@ -2,109 +2,127 @@
   <h3 class="ui header">{{t 'Tickets'}}</h3>
 </div>
 <div class="ui segment">
-  <form class="ui form">
-    <table class="ui very basic compact table" style="margin-bottom: 0">
-      <thead class="full-width">
-        <tr>
-          <th>{{t 'Type'}}</th>
-          <th class="four wide">{{t 'Sales Ends'}}</th>
-          <th class="ui two wide">{{t 'Price'}}</th>
-          <th class="one wide">{{t 'Quantity'}}</th>
-          <th class="ui right aligned two wide">{{t 'Subtotal'}}</th>
-        </tr>
-      </thead>
-      <tbody>
-        {{#each tickets as |ticket|}}
+  {{#if isTicketingEnabled}}
```

Comment:
This shouldn't be done here. `ticket-list` component is specifically for showing a list of tickets. Nothing else. Do the check and use this component only when there are tickets available. Revert this change from this file please.